apps/expo-payments/android/app/google-services.json

```diff
@@ -1,50 +0,0 @@
-{
-  "project_info": {
-    "project_id": "exponent-5dd6d",
-    "project_number": "367315174693",
-    "name": "Exponent Play Services"
-  },
-  "client": [
-    {
-      "client_info": {
-        "mobilesdk_app_id": "1:367315174693:android:f968efbb41d1fa7a",
```

Comment:
Do you know what is this project and whether the "Custom Expo Development" Firebase project is configured appropriately for In App Purchases>

Reply:
I don't there is any setup required in Firebase, this is just the services file that was created after ejecting to ExpoKit. The main concern was reusing the iOS payments as they can be more cumbersome to setup.

Reply:
The Custom Expo Development project is the old payments project. It’s just renamed but the application ID is the same as the payments app ID.